### DISCUSSION TITLE
AutoForm async extension proposal

### DISCUSSION THREAD

**Discussion by polygonwood** [meteor/meteor#12711] [Feature Requests]
Having little experience in contributing, I'm starting this discussion. While slowly adapting my major application to Meteor 3.0, I have added some new formtypes to AutoForm to make async calls in the submit loop.
I have added this to the branch **async** of the fork [autoform-async](https://github.com/polygonwood/meteor-autoform.git) .
This works for asynchronous methods or inserts. This does not (yet) support asynchronous hooks (rare in my application).
Who can comment on this ?

**Comment by harryadel** [comment]
The man the myth the legend @jankapunkt

**Reply by jankapunkt** [reply]
Hi @polygonwood let's continue discussion in the Autoform repository: https://github.com/Meteor-Community-Packages/meteor-autoform

I would be more than happy for any kickoff towards async Autoform.

**Reply by polygonwood** [reply]
Hi @jankapunkt 
the last activity on meteor-autoform dates from March, so not sure if posting there will get any attention, but I will create an issue there
Polygonwood

**Reply by jankapunkt** [reply]
No worries, it's under our hood, mostly maintained by myself but I first had to get the migration for simple schema on the right path before moving on with Autoform

**Reply by polygonwood** [reply]
OK i have added a discussion on async support for autoform  on [https://github.com/Meteor-Community-Packages/meteor-autoform](https://github.com/Meteor-Community-Packages/meteor-autoform)

**Comment by StorytellerCZ** [comment]
Closing this discussion as this is to be discussed at the appropriate place: https://github.com/Meteor-Community-Packages/meteor-autoform
